test/com/facebook/buck/android/testdata/android_project/kotlin/com/sample/lib/BUCK.fixture

```diff
@@ -0,0 +1,35 @@
+android_library(
+  name = 'lib',
+  srcs = glob(['*.kt']),
+  language = 'KOTLIN',
```

Comment:
We should test this but my feeling is java_library (and by association android_library before #913) expected all of it's sources to be java files. I've made the languages param optional in the latest update to this PR so we can see what that would be like. After testing the old behavior of java/scala we should decide what to do.